File: Day 58/Classwork/Classwork.py
string = lambda x:x[::-1].capitalize()
print(string("hello"))
print(string("giorgi"))
print(string("jioshvili"))

print((lambda x: x * 2)(3))

list0 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
print(list(map(lambda x: x**2, list0)))

File: Day 140/Classwork/Classwork.js
const eventsEmitter = require('events');
const myEmitter = new eventsEmitter();

function listener(info) {
    console.log(info);
}

myEmitter.on('welcome', listener);
myEmitter.emit('welcome', 'Hello');


const mybuffer = Buffer.from("Giorgi");

process.stdin.on("data", (input) => {
    const combinedBuffer = Buffer.concat([input, mybuffer]);

    const result = combinedBuffer.toString();
    process.stdout.write(result);

    process.exit();
});